components/raftstore-v2/src/batch/store.rs

```diff
@@ -182,6 +214,10 @@ struct StorePollerBuilder<EK: KvEngine, ER: RaftEngine, T> {
     engine: ER,
     tablet_factory: Arc<dyn TabletFactory<EK>>,
     trans: T,
+    router: StoreRouter<EK, ER>,
+    log_fetch_scheduler: Scheduler<RaftlogFetchTask>,
+    write_senders: Vec<Sender<WriteMsg<EK, ER>>>,
+    io_reschedule_concurrent_count: Arc<AtomicUsize>,
```

Comment:
Could we be more consistent in variable ordering? E.g. how about move cfg in the same place (`cfg`, `io_reschedule_concurrent_count`)

Reply:
I don't get it. Why they should be moved together? `io_reschedule_concurrent_count` and `write_senders` are put together because they are all related to async write.

Reply:
Then these should be grouped with newlines around them, otherwise how would you expect a newcomer to read these and grasp the subtext you are trying to imply here?